virttest/utils_net.py

```diff
@@ -4889,3 +4890,21 @@ def check_class_rules(ifname, rule_id, bandwidth, expect_none=False):
         stacktrace.log_exc_info(sys.exc_info())
         return False
     return True
+
+
+def obtain_guest_ip_from_dhcp_leases(mac):
+    """
+    Obtaining the guest ip address from virsh-net-dhcp-leases command
+    :param: Mac address of the guest
+    :return: return ip-address if found for given mac in the
+             virsh-net-dhcp-leases default table, else return None
+    """
+    output = virsh.net_dhcp_leases("default")
```

Comment:
This function will get dhcp lease from network "default" only, which might not be the default setting for test vms, right?